src/main/java/org/broadinstitute/consent/http/db/mapper/DraftSubmissionInterfaceMapper.java

```diff
@@ -0,0 +1,87 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+import java.util.UUID;
+import org.broadinstitute.consent.http.exceptions.NoMatchingClassException;
+import org.broadinstitute.consent.http.models.DraftInterface;
+import org.broadinstitute.consent.http.models.User;
+import org.jdbi.v3.core.mapper.RowMapper;
+import org.jdbi.v3.core.statement.StatementContext;
+
+public class DraftSubmissionInterfaceMapper implements RowMapper<DraftInterface>, RowMapperHelper {
+
+  @Override
+  public DraftInterface map(ResultSet rs, StatementContext ctx) throws SQLException, NoMatchingClassException {
+    if (!hasColumn(rs, "schema_class")) {
+      throw new NoMatchingClassException("Missing class name.");
+    }
+
+    DraftInterface dsi;
+    String className = rs.getString("schema_class");
+
+    try {
+      dsi = (DraftInterface) Class.forName(className).getDeclaredConstructor().newInstance();
+    } catch (ClassNotFoundException | NoSuchMethodException | InvocationTargetException |
+             InstantiationException | IllegalAccessException classNotFoundException) {
+      throw new NoMatchingClassException(className);
+    }
+
+    if (hasColumn(rs, "name")) {
+      dsi.setName(rs.getString("name"));
+    }
```

Comment:
This is not new to this PR, but there code duplication in the rowmappers here. It looks like a common pattern is `if (hasColumn(rs, x)) { ...(rs.getString(x) ... }`. Combining these operations would let you write
```
  default void getString(ResultSet rs, String columnName, Consumer<String> useString) throws SQLException {
    if (hasColumn(rs, columnName)) {
      useString.accept(rs.getString(columnName));
    }
  }
...

    getString(rs, "name", dsi::setName);
```

We'd need to have a different method for each type, but it would end up being less code overall in the end, and avoid the need to duplicate the column name in each case.

Reply:
Agreed, but I think that's going to be a bit larger of a refactor than this is change is intending.  Could I make a ticket for this so the team can talk about this?

Reply:
I agree, please do add another ticket - it's a nice utility we could pretty easily add where needed, but definitely orthogonal to this PR.

Reply:
https://broadworkbench.atlassian.net/browse/DT-939?atlOrigin=eyJpIjoiOGE2MjdkYTU5MDk3NDA3YmI4MmM1MGQ1NjJlZjU2OWMiLCJwIjoiaiJ9